app/views/works/form.html.erb

```diff
@@ -4,6 +4,7 @@
   <% component.with_tab(label: t('works.edit.panes.files.tab_label'), tab_name: :files, selected: true) %>
   <% component.with_tab(label: t('works.edit.panes.title.tab_label'), tab_name: :title) %>
   <% component.with_tab(label: t('works.edit.panes.abstract.tab_label'), tab_name: :abstract) %>
+  <% component.with_tab(label: I18n.t('works.edit.panes.dates.tab_label'), tab_name: :dates) %>
```

Comment:
I don't think we want to invoke `I18n.t` anywhere in H3 if we can avoid it, because it cuts out the Rails i18n niceness (such as HTML formatting depending on key name). We should use `t()` in views and otherwise use `helpers.t` outside of views (controllers, components).

Reply:
Agree. That was a merge screw up.